articles/native-platforms/ios-objc/10-customizing-lock.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize the Lock widget UI.
+---
+
+<%= include('../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-ios-objc-sample',
+  path: '10-Customizing-Lock',
+  requirements: [
+    'CocoaPods 1.1.1',
+    'Version 8.2 (8C38)',
+    'iPhone 6 - iOS 10.2 (14C89)'
+  ]
+}) %>
+
+### Before Starting
+
+You need the [Lock](https://github.com/auth0/Lock.iOS-OSX) library integrated in your project. Make sure you've followed the [login tutorial](01-login.md) and you know how to present the login dialog.
+
+### 1. Create a Theme
+
+Lock UI can be customized by creating your own `A0Theme` instance:
+
+```objc
+#import <Lock/Lock.h>
+```
+
+```objc
+A0Theme *theme = [[A0Theme alloc] init];
+```
+
+### 2. Configure Your Theme
+
+Lock's widget UI is composed of several parts that can be customized.
+
+![Lock.png](/media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png)
+
+You can configure these type of properties:
+
+- **Color**: `UIColor` instance.
+- **Image**: By using a `String` representing the image's name.
+- **Font**: `UIFont` instance.
+
+For instance:
+
+```objc
+[theme registerColor: [UIColor redColor] forKey: A0ThemePrimaryButtonNormalColor];
+```
+So, for example, if you want to achieve something like this:
+
+<div class="phone-mockup"><img src="/media/articles/native-platforms/ios-swift/Custom-Lock-Widget-Screenshot.png" alt="Mobile example screenshot"/></div>
+
+You will need to customize several parts. This is how you do it:
+
+```objc
```

Comment:
It might be better to use an ordered list here and just indent the code blocks to stay inline under their list item, in place of the commented numbering inside each block.